Makefile

```diff
@@ -51,7 +51,7 @@ CONTROLLER_GEN_VER := v0.16.0
 CONTROLLER_GEN_BIN := controller-gen
 CONTROLLER_GEN := $(abspath $(TOOLS_BIN_DIR)/$(CONTROLLER_GEN_BIN)-$(CONTROLLER_GEN_VER))
 
-STATICCHECK_VER := 2025.1.1
+STATICCHECK_VER := master
```

Comment:
needed for

2025.1.1 is already the latest release version of staticcheck and is not compatible with 1.24.9

```sh
-: module requires at least go1.24.9, but Staticcheck was built with go1.24.6 (compile)
```

Reply:
And master works for 1.24.9?